test/development/basic/styled-components/pages/index.js

```diff
@@ -23,7 +23,11 @@ const Button = styled.a`
 export default function Home() {
   console.log('__render__')
   return (
-    <div>
+    <div
+      css={`
+        background: black;
+      `}
```

Comment:
Can we add an assertion to test that this is applied correctly in https://github.com/vercel/next.js/blob/canary/test/development/basic/styled-components.test.ts?

We should be able to do something like:
```
window.getComputedStyle(document.querySelector(selector)).background === 'rgb(0, 0, 0)'
```

Reply:
done